components/ConvertSteps/Converter.js

```diff
@@ -0,0 +1,195 @@
+import React, { useCallback, useEffect, useState } from 'react'
+import Divider from './Divider'
+import PropTypes from 'prop-types'
+import StepperLayout from './StepperLayout'
+import { useAllowance, useOpenOrder, useClaimOrder } from 'lib/web3-contracts'
+import Step from './Step'
+import { formatUnits } from 'lib/web3-utils'
+import { useTokenDecimals } from 'lib/web3-contracts'
+
+function Converter({ toAnj, amountSource, amountRecipient, handleReturnHome }) {
+  const checkAllowance = useAllowance()
```

Comment:
```suggestion
  const changeAllowance = useAllowance()
```

This is a bit clearer :)

Reply:
👍